### ISSUE TITLE
Node Just Hangs Indefinitely when Unable to Talk to Bitcoin Core

### ISSUE THREAD

**Issue by adamkrellenstein** [CounterpartyXCP/counterparty-core#1930]
```
Counterparty Core v10.2.0
Verbosity: 0
Quiet: False
Configuration File: /Users/adam/Library/Application Support/counterparty/server.conf
Counterparty Database: /Users/adam/Library/Application Support/Bitcoin/counterparty.db
Server Log: /Users/adam/Library/Logs/counterparty/server.log
API Access Log: /Users/adam/Library/Logs/counterparty/server.access.log

------------------------------ START ------------------------------

2024-06-13T17:17:22.944068000+02:00 - [    INFO] - Connecting to database... (SQLite 3.45.1.0)
2024-06-13T17:17:22.960422000+02:00 - [    INFO] - Running PRAGMA optimize...
2024-06-13T17:17:24.316232000+02:00 - [    INFO] - Starting API Server.
```

**Comment by adamkrellenstein** [comment]
Dependent on #1960

**Comment by warrenpuffett** [comment]
No longer relevant.
